pkg/util/azureclient/mgmt/insights/activitylogs.go

```diff
@@ -4,25 +4,38 @@ package insights
 // Licensed under the Apache License 2.0.
 
 import (
-	mgmtinsights "github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2018-03-01/insights"
 	"github.com/Azure/go-autorest/autorest"
 	"github.com/Azure/go-autorest/autorest/azure"
+
+	//"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2018-03-01/insights"
+	"github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/insights/insights"
 )
 
+/* TODO (dranders): Change back to the normal azure-sdk-for-go
+once https://github.com/Azure/azure-sdk-for-go/issues/14704 is completed, or the 2015-04-01 azure API version
+successfully finds the missing objects:
+
+	activityLog.OperationName.Value == "Microsoft.Compute/virtualMachines/redeploy/action"
+	https://github.com/Azure/ARO-RP/blob/master/test/e2e/adminapi_redeployvm.go#L64-L70
```

Comment:
For a permanent link use a commmit hash:

```
https://github.com/Azure/ARO-RP/blob/924235001fd72793cc9770426dd74821c1e36bce/test/e2e/adminapi_redeployvm.go#L64-L70
```

Otherwise line selection will be broken.

Reply:
Is this a requirement for this PR?

Reply:
Changed.